man/find.ip.sites.Rd

```diff
@@ -33,6 +33,6 @@ annotated intronic regions
 \examples{
 \dontrun{
 library(BSgenome.Mmusculus.UCSC.mm10)
-ip.mm10 <- t.generate.ip.mask('refdata-cellranger-mm10-1.2.0/genes/genes.gtf',Mmusculus,'mm10')
```

Comment:
`roxygen2` picks up a deprecated function (i.e., `t.generate.ip.mask`) and replace it with the correct one (i.e., `find.ip.sites`).